cases-SEP31/util/schema.js

```diff
@@ -35,104 +30,38 @@ const transactionSchema = {
           type: ["string", "null"],
           format: "date-time",
         },
+        stellar_account_id: {
+          type: ["string", "null"],
+        },
+        stellar_memo_type: {
+          type: ["string", "null"],
+          pattern: "text|id|hash",
+        },
+        stellar_memo: {
+          type: ["string", "null"],
+        },
         stellar_transaction_id: {
           type: ["string", "null"],
         },
         external_transaction_id: {
           type: ["string", "null"],
         },
-        message: {
+        required_info_message: {
           type: ["string", "null"],
         },
+        required_info_updates: {
+          type: ["object", "null"],
+        },
         refunded: {
           type: "boolean",
         },
       },
-      required: [
-        "id",
-        "kind",
-        "status",
-        "more_info_url",
-        "amount_in",
-        "amount_out",
-        "amount_fee",
-        "started_at",
-        "completed_at",
-        "stellar_transaction_id",
-        "refunded",
-      ],
+      required: ["id", "status"],
```

Comment:
`stellar_account_id`, `stellar_memo`, and `stellar_memo_type` are also required per the SEP.